code-challenges.test.js

```diff
@@ -26,26 +31,80 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// input: 3 individual number objects
+// output: return a message depicting whether or not the number inputted is divisible by 3
+
+// const divisibleByThree = (number) => {
+//     // use number as placeholder for the arguement that will be console logged
+//     if (number % 3 === 0 ){
+//         // use mod % op to check if number is divis by three
+//         return `${number} is divisible by three`
+//         // if true, return message above
+//     } else {
+//         return `${number} is not divisible by three`
+//         // if false, or all else, return message above
+//     }
+// }
+// pass objects through function by console logging into original const(divisibleByThree)
+// console.log(divisibleByThree(object1))
+// console.log(divisibleByThree(object2))
+// console.log(divisibleByThree(object3))
+
+
+
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+describe('arrAllCaps', () => {
+    it('takes in an array of words and returns an array with all the words capitalized.', () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+        // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+        expect(arrAllCaps(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+        expect(arrAllCaps(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+
+
+    })
+})
+    // ReferenceError: arrAllCaps is not defined
+
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+// Pseudo code:
+
+const arrAllCaps = (words) => {
+    // iterate through array using .map to create new array for each value changed
+    return words.map( word => word.charAt(0).toUpperCase() + word.slice(1))
+    // for each value, use charAt(0).toUpperCase() to capitilize first letter, or index [0]
+    // .slice to retain the rest of each value starting from the second letter, or index [1]
+}
+const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// create new var to store the values that get manipulated from the function
+const capsNouns1 = arrAllCaps(randomNouns1)
+const capsNouns2 = arrAllCaps(randomNouns2)
```

Comment:
These variables are just storing the output of the function call with the given argument, important to note that distinction.